models/models.js

```diff
@@ -105,11 +105,35 @@ const postNewComment = (article_id, author, body) => {
     });
 };
 
+const patchArticleVotesByID = (id, inc_votes) => {
+  if (inc_votes === undefined) {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1;`, [id])
+    .then((article) => {
+      if (!article.rows.length) {
+        return Promise.reject({ status: 404, msg: "Not found" });
+      }
```

Comment:
This is fine but you do have a similar model you've already written which does this exact functionality.